python/tvm/topi/random/kernel.py

```diff
@@ -20,6 +20,8 @@
 from ... import tir
 from ...tir import ir_builder
 
+import numpy as np
```

Comment:
probably better if we move the test function definition to the `test_topi_prng.py` file also

Reply:
I want people to be able to run this function themselves to check if their platform is supported. If I put it in the test file, you can't import and use it. You'd have to edit the test manually to include your platform.

Reply:
I see, hmm. That makes sense, and I'm not sure there's a better way to indicate to devs other than the warning comment, so looks good. Can you fix the lint error? something about import numpy ordering